packages/ui-extensions/src/surfaces/admin/api/customer-segmentation-template/customer-segmentation-template.ts

```diff
@@ -0,0 +1,46 @@
+import type {StandardApi} from '../standard/standard';
+import type {ExtensionPoint as AnyExtensionPoint} from '../../extension-points';
+
+export enum CustomerSegmentationFeature {
+  ProductsPurchasedOnTags,
+  AggregateFilters,
+}
+
+export interface CustomerSegmentationTemplateApi<
+  ExtensionPoint extends AnyExtensionPoint,
+> extends StandardApi<ExtensionPoint> {
+  /** List of enabled segmentation query language features. */
+  enabledFeatures: [CustomerSegmentationFeature];
+  /** Utilities for translating content according to the current `localization` of the admin. */
+  i18n: I18n;
```

Comment:
Should we create a shared interface with Checkout? We really only need `i18n.translate()` for now.

Reply:
We should definitely align in terms of API but IIRC this is currently **not** available in Admin.

Reply:
@jonathanhamel4 has a PoC for Admin